src/Tutor.Infrastructure/Database/Repositories/LearningUtilities/FeedbackDatabaseRepository.cs

```diff
@@ -14,12 +14,10 @@ public FeedbackDatabaseRepository(TutorContext dbContext)
     public void SaveEmotionsFeedback(EmotionsFeedback emotionsFeedback)
     {
         _dbContext.EmotionsFeedbacks.Attach(emotionsFeedback);
-        _dbContext.SaveChanges();
```

Comment:
To avoid missing anything, did you do a search for "SaveChanges" and ensured that it is only present in UoW?

Reply:
Yes, apart from EventStore repository, which is a different context that I didn't refactor.